addons/hr_recruitment_integration_monster/data/monster_currency_unit_data.xml

```diff
@@ -0,0 +1,412 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="monster_currency_USD" model="monster.currency.unit">
```

Comment:
Remove this whole model and add a monster_id field on res.currency model instead (already done in several odoo models if you wanna check).

Reply:
removed as well